internal/models/pubkey_model.go

```diff
@@ -56,3 +58,9 @@ func (pk *Pubkey) FindAwsFingerprint(ctx context.Context) string {
 
 	}
 }
+
+func (pk *Pubkey) GetPubkeyWithUsername(ctx context.Context) string {
+	pubkey := strings.TrimSpace(pk.Body)
+	parts := strings.Split(pubkey, " ")
+	return fmt.Sprintf("gcp-user:%s %s", parts[0], parts[1])
```

Comment:
This can still panics, can we either check the boundaries or not to add `parts[1]`.

Reply:
yeah, I think we are missing a unit test case where the key is missing the optional (comment) part of the key

Reply:
We do not care about the comment (column 2), I adding a gcp-user instead.
Either way I added a condition regarding the size of `parts`.